pkg/doc/verifiable/credential.go

```diff
@@ -263,24 +308,79 @@ func issuerToSerialize(vc *Credential) interface{} {
 	return vc.Issuer.ID
 }
 
-func validate(data []byte) error {
-	// Validate that the Verifiable Credential conforms to the serialization of the Verifiable Credential data model (https://w3c.github.io/vc-data-model/#example-1-a-simple-example-of-a-verifiable-credential)
+func validate(data []byte, schema *CredentialSchema, opts *credentialOpts) error {
+	// Validate that the Verifiable Credential conforms to the serialization of the Verifiable Credential data model
+	// (https://w3c.github.io/vc-data-model/#example-1-a-simple-example-of-a-verifiable-credential)
+	schemaLoader, err := getCredentialSchema(schema, opts)
```

Comment:
Note: I don't think `credentialSchema` is mandatory

Reply:
I had that impression as well.

Reply:
Yeah, that's because of incorrect naming of the function. I renamed it to `getSchemaLoader()` which reflects much better the purpose of this function. It uses default JSON schema by default and can bring custom credential schema if a) it's defined in Verifiable Credential; b) configuration allows to use it; c) the download of the schema is successful.